sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaSinkMetrics.java

```diff
@@ -71,6 +75,48 @@ public static Histogram createRPCLatencyHistogram(RpcMethod method, String topic
     return new DelegatingHistogram(metricName, buckets, false, true);
   }
 
+  /**
+   * Creates an Gauge metric to record per partition backlog. Metric will have name:
+   *
+   * <p>'EstimatedBacklogSize*topic_name:{topic};partitionId:{partitionId};'
+   *
+   * @param topic Kafka topic associated with this metric.
+   * @param partitionId partition id associated with this metric.
+   * @return Counter.
+   */
+  public static Gauge createBacklogGauge(String topic, int partitionId) {
+    return new DelegatingGauge(getMetricGaugeName(topic, partitionId), false, true);
+  }
+
+  /**
+   * Creates an Gauge metric to record per partition backlog. Metric will have name:
```

Comment:
Nit: an Gauge -> a {@link Gauge}, ". Metric will have name:" -> " with the name:"